### DISCUSSION TITLE
`!!!cmd`は誤記でしょうか？

### DISCUSSION THREAD

**Discussion by mendelssohnbach** [vscodejp/handson-hello-vscode-extension#11] [Q&A] [answered]
[Yeoman ── ソースコードを自動生成するツール](https://github.com/vscodejp/handson-hello-vscode-extension/blob/main/docs/00_prepare.md#yeoman--%E3%82%BD%E3%83%BC%E3%82%B9%E3%82%B3%E3%83%BC%E3%83%89%E3%82%92%E8%87%AA%E5%8B%95%E7%94%9F%E6%88%90%E3%81%99%E3%82%8B%E3%83%84%E3%83%BC%E3%83%AB)

ハンズオン開始まえですが質問を書かせていただきます。

```
!!!cmd
$ npm install -g yo generator-code
```

`!!!cmd`は誤記でしょうか？
私の環境では実行することができませんでした。

```
$ !!!cmd
bash: !!: event not found
```

実行しない状態で[コースB: 補完機能を実装しよう]まで進むことができました。

**Comment by Ikuyadeu** [comment, marked as answer]
@mendelssohnbach 早速進めていただきありがとうございます．

`cmd`は誤表記です．ただいま修正いたしました．
ご指摘いただきありがとうございます．

**Reply by mendelssohnbach** [reply]
@Ikuyadeu

お忙しい中、早速の対応ありがとうございます。
ハンズオンテキストの修正を確認しました。
